.cirrus.yml

```diff
@@ -26,17 +26,17 @@ freebsd_task:
   name: FreeBSD
 
   matrix:
-    - name: FreeBSD 13.0
+    - name: FreeBSD 13.1
       freebsd_instance:
-        image_family: freebsd-13-0
+        image_family: freebsd-13-1
     - name: FreeBSD 12.3
       freebsd_instance:
         image_family: freebsd-12-3
 
   env:
     CIRRUS_CLONE_DEPTH: 10
     CRYPTOGRAPHY_DONT_BUILD_RUST: 1
-    MAKE_FLAGS: -j 2
+    MAKE_FLAGS: -j ${nporc}
```

Comment:
Isn't this misspelled and should rather be `nproc`?

Reply:
You right, will fix it later